Parse/src/main/java/com/parse/ParsePush.java

```diff
@@ -422,6 +442,14 @@ public void clearExpiration() {
   }
 
   /**
+   * Sets a UNIX epoch timestamp at which this notification should be delivered, in seconds (UTC).
+   * Scheduled time can not be in the past and must be at most two weeks in the future.
+   */
+  public void setPushTime(long time) {
```

Comment:
Could it be `public void setPushTime(Date date)`? 
(`pushTime = date.getTime()/1000`)

or change the time unit to milliseconds to match Java's timestamp
(`pushTime = time/1000`)

Because unit of UNIX epoch timestamp is seconds, but developers probably using `Date.getTime()` or `System.currentTimeMillis()` to get timestamp (unit is milliseconds), then they will fail if not converted.